### DISCUSSION TITLE
Can I reuse the connection when I duplicate a tab?

### DISCUSSION THREAD

**Discussion by hefnrh** [Eugeny/tabby#5242] [Q&A] [answered]
Like putty: connection->ssh->Sharing an SSH connection between PuTTY tools->Share connections if possible.
Because the server uses RSA token as password, it changes every 30 seconds.

**Comment by Eugeny** [comment, marked as answer]
It's not possible yet, see #4795

**Comment by lawweiliang** [comment]
@hefnrh  
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
